app/static/js/main.js

```diff
@@ -391,9 +391,13 @@ if (!document.getElementsByClassName) {
 }
 
 addEvent("DOMContentLoaded", document, function(e) {
-    $(".timeago").timeago();
+    var $times = $('.timeago');
+    for (let i=0; i<$times.length; i++) {
+        var times_i = $times[i];
+        times_i.innerText = jQuery.timeago(times_i.title);
+    }
```

Comment:
```suggestion
    var times = $('.timeago');
    for (let i = 0; i < times.length; i++) {
        times[i].innerText = jQuery.timeago(times[i].title);
    }
```
Some small nitpicks, nothing much else